src/bloaty.cc

```diff
@@ -708,12 +810,30 @@ void RollupOutput::PrettyPrintRow(const RollupRow& row, size_t indent,
 
   if (ShowFile(options)) {
     *out << PercentString(row.filepercent, diff_mode_) << " "
-         << SiPrint(row.filesize, diff_mode_) << " ";
+         << SiPrint(row.size.file, diff_mode_) << " ";
+    
+    if (row.capacity.file) {
+      if (row.capacity_origin & CapacityOrigin::kFileSet) {
+        capacity_total_ = false;
```

Comment:
I don't think that this should be set here, as this is a print function. Functions which output should typically be pure and not modify state (hence the `const` specifier).

I think a suitable place to set this flag would be during parsing of the command-line options and proto file.